data/presets/natural/rock.json

```diff
@@ -1,7 +1,15 @@
 {
     "icon": "temaki-boulder2",
     "fields": [
-        "name"
+        "name",
+        "height",
+        "geological_outcrop"
+    ],
+    "moreFields": [
+        "width",
+        "diameter",
+        "circumference",
```

Comment:
This field is currently called “Circumference (at Breast Height)” because the `natural=tree` tagging scheme derives `circumference` from a diameter at breast height (DBH) measurement. (Go figure.) We’ll need to split out a different field for circumferences of other things, like rocks, for which “breast height” would be baffling.

/ref #455

Reply:
Didn't realize it was that specific - I think there are already petrified trees on the map as natural=rock so it's not totally out of the question but diameter will do.